model_pkgs/mlflow_pkg/package.py

```diff
@@ -0,0 +1,33 @@
+import mlflow.pyfunc
+from model_code.tf_mlflow_model import TranscriptformerMLflowModel
+
+variants = [
+    {
+        "name": "tf_sapiens",
+        "checkpoint": "/checkpoints/tf_sapiens",
+        "embedding": "/checkpoints/all_embeddings/homo_sapiens_gene.h5",
+    },
+    {
+        "name": "tf_exemplar",
+        "checkpoint": "/checkpoints/tf_exemplar",
+        "embedding": "/checkpoints/all_embeddings/exemplar.h5",
+    },
+    {
+        "name": "tf_metazoa",
+        "checkpoint": "/checkpoints/tf_metazoa",
+        "embedding": "/checkpoints/all_embeddings/metazoa.h5",
+    },
+]
+
+for variant in variants:
```

Comment:
oh, perfect. this is exactly what we want--one mlflow model per variant (eliminating the "variant" param)